### PR TITLE
20180108 Add images#create, sounds#show, sounds#create

### PR DESCRIPTION
**Pull request by avolkovi**

### CODE REVIEW

File: source/includes/assets/_sounds.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+* `mp3`
+* `mp4`
+* `mpa`
+* `mpg`
+* `mpga`
+* `wav`
+* `ogg`
+* `mid`
+* `midi`
+* `kar`
```
Comment (strom2357) [inline source/includes/assets/_sounds.md:135]: ...do we really support all of these?

Comment (avolkovi) [reply source/includes/assets/_sounds.md:135]: it's basically anything that we can shove into ~lame~ sox, it'll convert everything to mp3

Comment (avolkovi) [reply source/includes/assets/_sounds.md:135]: see: https://github.com/TrevorHinesley/carrierwave-audio

Comment (avolkovi) [reply source/includes/assets/_sounds.md:135]: here's the full list we could theoretically support: http://sox.sourceforge.net/soxformat.html

File: source/includes/courses/_users.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    -H "Authorization: Bearer <BEARER_TOKEN>"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "1158898",
```
Comment (strom2357) [inline source/includes/courses/_users.md:17]: TIL Kyle... is Abe Lincoln...

Comment (kwstewar) [reply source/includes/courses/_users.md:17]: reincarnated on his bday

File: source/includes/assets/_sounds.md (hunk 1 above)
Comment (strom2357) [reply source/includes/assets/_sounds.md:135]: Sweet, matches our whitelist, just double checking

**Comment by strom2357** [comment]
It won't let me change labels but looks fine to me

File: source/includes/courses/_users.md (hunk 2 above)
Comment (zaguios) [reply source/includes/courses/_users.md:17]: Did anyone read the example JSON responses on the last PR?

Comment (strom2357) [reply source/includes/courses/_users.md:17]: Oh boy, nope, am now, gonna have to sign up for Genghis Khan's World Domination course

**Comment by avolkovi** [comment]
@strom2357 try again, I fiddled with the permissions

**Pull request merged**